concise-mid-tag.cddl

```diff
@@ -182,11 +182,12 @@ operational-flags = &(
 )
 
 raw-value-group = (
-  comid.raw-value => raw-value-type
+  comid.raw-value => $raw-value-type-choice
   ? comid.raw-value-mask => raw-value-mask-type
```

Comment:
I have always found a bit "untidy" that raw-value and mask are treated as a group rather than as a true type, but this ship has sailed and I've done nothing to prevent that, so I should just shut up :-)
That said, it seems to me that adding a tag to the mask is not going to add any benefit because the coupling needs to be defined in prose anyways.

Reply:
I don't think I get it. Do you mean the coupling between the tagged raw-value and the raw-value-mask?

Reply:
>  Do you mean the coupling between the tagged raw-value and the raw-value-mask?

yes, I these go hand in hand, so tagging one will necessarily influence the semantics of the other.